apps/src/templates/levelSummary/SummaryResponses.jsx

```diff
@@ -143,13 +142,15 @@ const SummaryResponses = ({
           {/* Correct answer toggle */}
           {showAnswerToggle && (
             <div className={styles.toggleContainer}>
-              <ToggleSwitch
-                isToggledOn={showCorrectAnswer}
-                onToggle={() => {
+              <Toggle
+                onChange={() => {
                   setShowCorrectAnswer(!showCorrectAnswer);
                 }}
+                checked={showCorrectAnswer}
                 label={i18n.showAnswer()}
-                expands="summary-correct-answer"
+                position={'right'}
```

Comment:
Moved to the right to match other toggle on free response. @MollyPeredo to approve this change.

Reply:
Approved. Thank you for updating this!